### PR TITLE
Add missing props

### PR DESCRIPTION
**Pull request by marandaneto**
## :bulb: Motivation and Context
Some props were not synced down to native SDKs and features enabled by default could not be turned off.

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] I added tests to verify changes
- [x] No new PII added or SDK only sends newly added PII if `sendDefaultPii` is enabled
- [ ] I updated the docs if needed
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

**Review by brustolin — APPROVED** [review]

File: flutter/ios/Classes/SentryFlutterPluginApple.swift [hunk 1]
```diff
@@ -313,18 +313,12 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
             options.dist = dist
         }
 
-        if let enableAutoNativeBreadcrumbs = arguments["enableAutoNativeBreadcrumbs"] as? Bool,
-           enableAutoNativeBreadcrumbs == false {
-            options.integrations = options.integrations?.filter { (name) -> Bool in
-                name != "SentryAutoBreadcrumbTrackingIntegration"
-            }
+        if let enableAutoNativeBreadcrumbs = arguments["enableAutoNativeBreadcrumbs"] as? Bool {
+            options.enableAutoBreadcrumbTracking = false
```
Comment (marandaneto) [inline flutter/ios/Classes/SentryFlutterPluginApple.swift:317]: @brustolin can you confirm that `enableAutoBreadcrumbTracking` is the same as filtering `SentryAutoBreadcrumbTrackingIntegration`? the boolean `enableAutoBreadcrumbTracking` didn't exist before.

File: flutter/ios/Classes/SentryFlutterPluginApple.swift [hunk 2]
```diff
@@ -313,18 +313,12 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
...
         }
 
-        if let enableNativeCrashHandling = arguments["enableNativeCrashHandling"] as? Bool,
-           enableNativeCrashHandling == false {
-            options.integrations = options.integrations?.filter { (name) -> Bool in
-                name != "SentryCrashIntegration"
-            }
+        if let enableNativeCrashHandling = arguments["enableNativeCrashHandling"] as? Bool {
+            options.enableCrashHandler = false
```
Comment (marandaneto) [inline flutter/ios/Classes/SentryFlutterPluginApple.swift:321]: @brustolin can you confirm that the `enableCrashHandler` flag is the same as filtering `SentryCrashIntegration`? this flag didn't exist before.

File: flutter/ios/Classes/SentryFlutterPluginApple.swift (hunk 1 above)
Comment (brustolin) [reply flutter/ios/Classes/SentryFlutterPluginApple.swift:317]: Yes this is correct. `enableAutoBreadcrumbTracking` enabled `SentryAutoBreadcrumbTrackingIntegration`

File: flutter/ios/Classes/SentryFlutterPluginApple.swift (hunk 2 above)
Comment (brustolin) [reply flutter/ios/Classes/SentryFlutterPluginApple.swift:321]: Correct. `enableCrashHandler` is related to `SentryCrashIntegration`

**Pull request merged**